sat/cli/bootsys/parser.py

```diff
@@ -267,3 +280,4 @@ def add_bootsys_subparser(subparsers):
 
     _add_bootsys_shutdown_subparser(actions_subparsers)
     _add_bootsys_boot_subparser(actions_subparsers)
+    _add_bootsys_reboot_subparser(actions_subparsers)
```

Comment:
Now that there's a reboot action, I would suggest altering lines 266-268 as follows (similar to the man page summary update):

```
        description='Perform boot, shutdown, or reboot actions on the system.',
```